### DISCUSSION TITLE
OPFS under linux (webkit2gtk 4.1) possible?

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#11973] [Q&A]
Did not find any info if it works with 4.1.
Are capabilities needed to run it?

**Comment by FabianLars** [comment]
Did you try it? Since it's relatively new (in browser time) it may totally be possible that it's simply not supported by webkitgtk or even webkit asa whole yet.

Capabilities do not affect webviews native functionality.

Edit: you may get better answers by asking the webkitgtk maintainers instead of us :)
